serving/accessing-metrics.md

```diff
@@ -3,9 +3,11 @@
 You access metrics through the [Grafana](https://grafana.com/) UI. Grafana is
 the visualization tool for [Prometheus](https://prometheus.io/).
 
+## Grafana
+
 1. To open Grafana, enter the following command:
 
-```
+```shell
 kubectl port-forward --namespace knative-monitoring $(kubectl get pods --namespace knative-monitoring --selector=app=grafana --output=jsonpath="{.items..metadata.name}") 3000
```

Comment:
Is it possible to split this into multiple lines (using "\") for readability? As well as the prometheus one?